tests/frontend/test_io.py

```diff
@@ -66,7 +66,7 @@ def prog():
         # measurement
         ops.MeasureX | q[0]
         ops.MeasureHomodyne(0.43, select=0.32) | q[2]
-        ops.MeasureHomodyne(phi=0.43, select=0.32) | q[2]
```

Comment:
@thisac does this line no longer work?

Reply:
I'm _pretty_ sure it would be fine to leave as it was. Likely overdid it a bit in my updates to the test suite.